source/develop/release-management/features/metrics/metrics-store-installation.html.md

```diff
@@ -122,63 +84,101 @@ This procedure will define the curator pod so that it deletes metrics indexes th
+
+  Environment name. Can be used to identify data collected in a single central
+  store sent from more than one oVirt engine.
+
+- `fluentd_elasticsearch_host:` (required - no default value)
+
+  Address or hostname (FQDN) of the Elasticsearch server host.
+
+- `ovirt_env_uuid_metrics:` (required - no default value)
+
+  UUID of the project/namespace used to store metrics records.
+  This is used to construct the index name in Elasticsearch.
+  For example, if you have ovirt_env_name: myenvname,
+  then in logging OpenShift you will have a project named ovirt-metrics-myenvname.
+  You need to get the UUID of this project like this:
+  oc get project ovirt-metrics-myenvname -o jsonpath='{.metadata.uid}'
 
-        # cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine/config.yml
+- `ovirt_env_uuid_logs:` (required - no default value)
 
-5. Update the file /etc/ovirt-engine-metrics/config.yml located on the engine machine, with the following lines:
+  UUID of the project/namespace used to store log records.
+  This is used to construct the index name in Elasticsearch.
+  For example, if you have ovirt_env_name: myenvname,
+  then in logging OpenShift you will have a project named ovirt-logs-myenvname.
+  You need to get the UUID of this project like this:
+  oc get project ovirt-logs-myenvname -o jsonpath='{.metadata.uid}'
 
-        # vi /etc/ovirt-engine-metrics/config.yml.example
+- `fluentd_elasticsearch_ca_cert_path:` (required - no default value)
 
-    Replace the example values with your specific environment details:
+  The path to the file containing the CA certificate of the CA that issued
+  the Elasticsearch SSL server cert.
+  Get it from the logging OpenShift machine like this:
+  oc get secret logging-fluentd --template='{{index .data "ca"}}' | base64 -d > fluentd-ca
+  and copy the file to the engine machine.
 
-     * "fluentd-server.example.com" - The fully qualified domain name of the metrics store machine
+- `fluentd_elasticsearch_client_cert_path:` (required - no default value)
 
-     * "my_shared_key" - The shared key configured in fluentd on the metrics store machine.
+  The path to the file containing the SSL client certificate to use
+  with certificate authentication to Elasticsearch.
+  Get it from the logging OpenShift machine like this:
+  oc get secret logging-fluentd --template='{{index .data "cert"}}' | base64 -d > fluentd-cert
+  and copy the file to the engine machine.
 
-     * "/path/to/fluentd_ca_cert.pem" - The path to the fluentd CA certificate
+- `fluentd_elasticsearch_client_key_path:` (required - no default value)
 
-     * "ovirt_env_name" - **Use the same name you configured earlier**.
+  The path to the file containing the SSL client key to use
+  with certificate authentication to Elasticsearch.
+  Get it from the logging OpenShift machine like this:
+  oc get secret logging-fluentd --template='{{index .data "key"}}' | base64 -d > fluentd-key
+  and copy the file to the engine machine.
```

Comment:
167-171 - revisit. not clear

Reply:
They need to do this operation on the netrics store machine  to get the file and copy it to the engine machine. Then state here the path to this file.